aio/content/examples/observables/src/creating.spec.ts

```diff
@@ -1,12 +1,16 @@
 import { docRegionFromEvent, docRegionSubscriber } from './creating';
-import { cold } from 'jasmine-marbles';
 
 describe('observables', () => {
   it('should create an observable using the constructor', () => {
-    const console = { log: jasmine.createSpy };
+    const consoleLogSpy  = spyOn(console , 'log');
```

Comment:
Generally, it is preferable to use a spy object (instead of spying on the real `console.log()` (in case some other pieve of code wants to `console.log()` something):

```suggestion
    const console = {log: jasmine.createSpy('log')};
```